times_reader/transforms.py

```diff
@@ -1780,15 +1783,35 @@ def has_no_wildcards(list):
                 result.append(replace(table, dataframe=df, tag=datatypes.Tag.fi_t))
             else:
                 # wildcard expansion will happen later
+                # Handle named columns in data area:
                 if table.tag == datatypes.Tag.tfm_ins_ts:
-                    # ~TFM_INS-TS: Regions should be specified in a column with header=Region and columns in data area are YEARS
+                    # ~TFM_INS-TS: columns in data area are YEARS
```

Comment:
Looks like we already have code to handle `TFM_INS-TS`

Reply:
Yes, we do. :-) The idea for #120 is to handle the variants before proceeding with other transforms. I.e. this way we would reduce the number of `elif` statetements and improve readability of the code.

Reply:
Ah yes, I was coming to the same conclusion myself! Let me see if I can refactor this code to make it simpler.